tests/dhcp_relay/test_dhcp_relay.py

```diff
@@ -347,12 +362,17 @@ def test_dhcp_relay_default(ptfhost, dut_dhcp_relay_data, validate_dut_routes_ex
             if not skip_dhcpmon:
                 time.sleep(18)      # dhcpmon debug counter prints every 18 seconds
                 loganalyzer.analyze(marker)
+                if testing_mode == DUAL_TOR_MODE:
+                    loganalyzer_standby.analyze(marker_standby)
     except LogAnalyzerError as err:
         logger.error("Unable to find expected log in syslog")
         raise err
 
     # Clean up - Restart DHCP relay service on DUT to recover original dhcpmon setting
     restart_dhcp_service(duthost)
+    if testing_mode == DUAL_TOR_MODE:
+        restart_dhcp_service(standby_duthost)
+        pytest_assert(wait_until(120, 5, 0, check_interface_status, standby_duthost))
     pytest_assert(wait_until(120, 5, 0, check_interface_status, duthost))
```

Comment:
@kellyyeh should we add else here? Otherwise, check_interface_status will run twice on single TOR?

Reply:
It will run one more time after dhcpmon test because we're restarting dhcp_relay docker to
On dualtor mode it will run once on standby and once on active